### ISSUE TITLE
AssetSync has moved / revamp

### ISSUE THREAD

**Issue by davidjrice** [AssetSync/asset_sync#316]
So, AssetSync has moved. Apologies for the tumbleweeds and lack of response on issues of late. The community has been doing a good job of keeping on top of things with plenty of network activity. Thanks for that.

* I hope to personally put in a good bit of time in the coming weeks updating, refactoring, documenting and pulling in a lot of the suggestions and pull requests.
* Have recently returned to freelance work, just wondering if people are cool if I were to setup something like gittip? I dunno. Just an idea to hopefully inspire me to do more, through increased guilt, or something. lol.
* Also want to invite anyone who has ever contributed a pull request, to fire me a message. I'll review and potentially give commit bits to a few of those people, to help get things moving even more.

Anything else I should be doing? Let me know.

Best,
Dave

**Comment by bruno** [comment]
Thank you @davidjrice ... I'd definitely contribute to a gittip or alike to keep on supporting `asset_sync`!

**Comment by mbhnyc** [comment]
:+1: happy to see this picked up :dancer:

**Comment by davidjrice** [comment]
Thanks folks :) Don’t worry. I’ll hopefully get a chance to get stuck in to some things next week.

**Comment by mintuhouse** [comment]
@paulrnash may be interested

**Comment by stepchud** [comment]
What does it mean that AssetSync is "moved"? Does it mean this repo isn't the authoritative version anymore? What's the public gem version drawn from? As of this writing, the last commits to master were over a year ago, so it seems it has completely stalled. The gzip compression can be merged (or at least some feedback about it), it works and solves the problem => https://github.com/AssetSync/asset_sync/pull/310

**Comment by PikachuEXE** [comment]
Any plan on adding some collaborators?

**Comment by davidjrice** [comment]
Yep. If anyone is volunteering! I will consider adding them. So far no-one!

Sorry I still haven't got to restarting this. I'm really not sure the direction to take, I've been waiting till an opportunity to work on a project with asset_sync installed comes up to address these things as pretty much all the projects I'm working on at present, that pay the bills, do not have any assets served from rails.

**Comment by PikachuEXE** [comment]
I can volunteer to review **some** easy to understand Pull Requests like #315 
But that's all I can do with my time

I still got to maintain `i18n-js` (I am the only collaborator there :S)

I guess a "test project" should be created for debugging some issues
What bills need to be payed ?_?

**Comment by davidjrice** [comment]
Well then, just added the first contributor. Welcome @PikachuEXE 

Feel free to start preparing a new release in a branch if you want help reviewing.

There is a test project or two somewhere for older rails versions. Plus a good bit of tests in the repo however yeah, some sort of better integration test would be good.

Oh, just those pesky personal bills o.O

**Comment by PikachuEXE** [comment]
Should I commit/merge things into `master` or another branch?
Also what would be the release process? (now that the project got collborators)

**Comment by davidjrice** [comment]
Yeah. Branch would be great. I can help review then easily. Should add
contib guidelines too, yep

**Comment by PikachuEXE** [comment]
What about Pull Requests?
Should I pull them myself?

Release should also prepared by branch + PR?

**Comment by davidjrice** [comment]
Not sure what to do best long term, tho preparing pull request into a branch for testing prior to merging is always a good idea! Feel free to prepare anything into individual testing branches and and/or collate those testing branches into a release branch, that’s generally how I’d do it.

**Comment by PikachuEXE** [comment]
@davidjrice 
I just saw #324 and I am thinking it would be a good example (since it's small) to know what actually should be done.

For my own gems, I will merge it into `master` by pressing the merge button (after it's reviewed)
And that's the default behaviour for starting Pull Request

But you mentioned that "preparing pull request into a branch for testing prior to merging is always a good idea!" and I don't understand what it means.
Maybe you can explain more with that example?

**Comment by mtparet** [comment]
@davidjrice @davidjrice I'm interessed too. I'm committed to support further Rails versions and minor changes like https://github.com/AssetSync/asset_sync/pull/300 https://github.com/AssetSync/asset_sync/pull/294 etc..

**Comment by mbhnyc** [comment]
I think the recommendation is to stop using gems like this and switch to site-based CDN, cloudfront can now pull edge assets from a server of your choosing (i.e. your rails app), and automatically gzip them. That's what i'm doing, which of course has the benefit of speeding up deployments by a fair amount, and not using S3 to store assets that exist on your server anyway.

Better!!

**Comment by PikachuEXE** [comment]
@mbhnyc 
I guess the point of using this gem is to ensure the old assets are still available for a long time
e.g. images in old emails

Some applications require it, some don't
So it depends

**Comment by PikachuEXE** [comment]
@davidjrice 
I guess the fastest way for us to learn the process is to demonstrate how you you do it.
Any thoughts or schedule?
